.ci/scripts/install_windows.bat

```diff
@@ -15,17 +15,18 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 rem ============================================================================
 
+rem %1 - url to download
+rem %2 - components to download (following oneapi standards, default: default)
+rem %3 - install directory (default: C:\temp\oneapi\)
+
 set URL=%1
-set COMPONENTS=%2
+if "%2"=="" (set COMPONENTS=default) else (set COMPONENTS=%2)
```

Comment:
where is set the default components of toolkit?

Reply:
![image](https://github.com/user-attachments/assets/49aa7681-4301-48b2-9adf-747ca9f89747)